localstack-core/localstack/services/lambda_/provider.py

```diff
@@ -1541,14 +1541,19 @@ def get_function(
                 RepositoryType=image.repository_type,
                 ResolvedImageUri=image.resolved_image_uri,
             )
+        concurrency = None
+        if fn.reserved_concurrent_executions:
+            concurrency = GetFunctionConcurrencyResponse(
+                ReservedConcurrentExecutions=fn.reserved_concurrent_executions
+            )
 
         return GetFunctionResponse(
             Configuration=api_utils.map_config_out(
                 version, return_qualified_arn=bool(qualifier), alias_name=alias_name
             ),
             Code=code_location,  # TODO
             **additional_fields,
-            # Concurrency={},  # TODO
+            Concurrency=concurrency,
```

Comment:
Could we move this one line up? I think we generally do **fields in the end in lambda.